src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -5332,6 +5341,9 @@ void StorageReplicatedMergeTree::shutdown(bool)
     if (shutdown_called.exchange(true))
         return;
 
+    const auto storage_name = getStorageID().getNameForLogs();
+    LOG_TRACE(log, "Shutdown started, table={}", storage_name);
+
```

Comment:
```suggestion
    LOG_TRACE(log, "Shutdown started");
```
Doesn't the log already contain the table name and uuid?

Reply:
ok